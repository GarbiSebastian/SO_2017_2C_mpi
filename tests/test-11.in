load corpus-0 corpus-1 corpus-2 corpus-3 corpus-4 corpus-5 corpus-6 corpus-7 corpus-8 corpus-9
maximum
q
