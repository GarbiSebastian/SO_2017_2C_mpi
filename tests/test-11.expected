Comandos disponibles:
  load <arch_1> <arch_2> ... <arch_n>
  addAndInc <string>
  member <string>
  maximum
  q|quit
> La listá esta procesada
> El máximo es <volvia,4>
> Finalizamos ejecución
